src/Panels/HostGamePanel.cs

```diff
@@ -111,6 +111,15 @@ public override void Start()
             };
         }
 
+        public override void Update()
```

Comment:
Add this logic also to the ConnectionPanel (which is opened instead of the HostGamePanel when the server is running)